modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/java/com/liferay/frontend/data/set/admin/web/internal/display/context/FDSAdminItemSelectorDisplayContext.java

```diff
@@ -41,6 +48,18 @@ public long getClassNameId() {
 		return PortalUtil.getClassNameId(getClassName());
 	}
 
+	public String getItemSelectedEventName() {
+		return _itemSelectedEventName;
```

Comment:
Tried `itemSelectedEventName`, also provided in [itemSelector](https://github.com/liferay-frontend/liferay-portal/pull/4362/files#diff-b094c5f25620284eeb36a69b9802d581e15de4acf2397664ca9b1ff1dd398542R69) to use as eventListener.